src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -269,16 +269,22 @@ typedef clock_num_t clock_handle_t;
  * frequency to be specified, and will set the clock divider to achieve the exact or higher frequency
  * achievable, with the maximum being the src_freq.
  *
- * Note: That the clock hardware only support divisors of exactly 1 or 2.0->65535.0
+ * \if rp2350_specific
+ * Note: That the RP2350 clock hardware supports divisors from 1.0->65535.0 in steps of 1/65536
```

Comment:
Looking at e.g. https://www.raspberrypi.com/documentation/pico-sdk/hardware.html it seems that "Note: That the..." should just be "Note: The..." ?